wikisrv.fly.toml

```diff
@@ -0,0 +1,23 @@
+# fly.toml app configuration file generated for wiki-srv on 2023-12-13T14:28:57-08:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "wiki-srv"
```

Comment:
in total I'll create:

* wiki-srv
* todo-srv
* mysql <--- will create a single db that is shared so I stay in the free tier